resources/layout/_head.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Html\Html;
+
```

Comment:
We need to declare all the variables used i.e.

```php
/* @var \App\LayoutParameters $layoutParameters */
/* @var string $csrf */
```

Reply:
done.